lua/autosave/modules/autocmds.lua

```diff
@@ -127,14 +148,20 @@ local function parse_events()
 end
 
 function M.load_autocommands()
+    if opts["debounce_delay"] == false then
```

Comment:
Wow, I really like the idea. Turning it to 140 mlsc by default would be a great idea too. Apart from the small logic issue above (which I can fix locally, don't worry about that), everything looks great. I personally have never noticed any delays, probably because I'm not using any "kind of 'live' server that monitors changes on the file system and reloads webpages etc.", but seems like a fair enough reason to implement such a  feature.

Good job and thanks for the PR!

Reply:
I also noticed you didn't update the doc. Don't worry about that, I'll do that now too :+1:

Reply:
A missed this, changed from `false` to `0` at the last minute and forgot this one.